src/polychord/mpi_utils.F90

```diff
@@ -164,7 +164,7 @@ subroutine initialise_mpi(feedback)
             call MPI_INIT(mpierror)
             finalize = .true.
         else
-            if(feedback >= normal_fb) write(*,'("PolyChord: MPI is already initilised, not initialising, and will not finalize")')
+            if(feedback >= normal_fb) write(*,*) "PolyChord: MPI is already initilised, not initialising, and will not finalize"
```

Comment:
For consistency with the rest of the code, I would prefer this to be the master version

Reply:
Sure. That line gave me a segmentation fault at some point but that might have been due to the general problems I had with the MPI runtime libraries.